cadc-pkg-server/src/main/java/org/opencadc/pkg/server/PackageRunner.java

```diff
@@ -204,20 +207,12 @@ private void doIt() {
             log.debug(job.getID() + ": EXECUTING -> COMPLETED [OK]");
 
         } catch (Throwable t) {
-            if (ThrowableUtil.isACause(t, InterruptedException.class)) {
-                try {
-                    ep = jobUpdater.setPhase(job.getID(), ExecutionPhase.QUEUED, ExecutionPhase.EXECUTING, new Date());
-
-                    if (!ExecutionPhase.ABORTED.equals(ep)) {
-                        return; // clean exit of aborted job
-                    }
-
-                } catch (Exception ex2) {
-                    log.error("failed to check job phase after InterruptedException", ex2);
-
-                }
+            try {
+                ep = jobUpdater.setPhase(job.getID(), job.getExecutionPhase(), ExecutionPhase.ERROR, new Date());
```

Comment:
`job.getExecutionPhase( )` is not reliably up-to-date (it's probably still SUSPENDED in this case)... I think you can hard-code EXECUTING here because logically nothing happens before SUSPENDED->EXECUTING or after EXECUTING->COMPETED.

there really should be a catch designed to handle those setPhase operations throwing a job-related exception, but because there is a `catch(Throwable)` your IDE  doesn't tell you about what those are... out of scope. Just hard code EXECUTING->ERROR

Also, for the sendError, it was using the two-arg method so that "unexpected failure" would be added to the output. Please revert to that.